components/Sidebar/panels/Queue/index.tsx

```diff
@@ -0,0 +1,37 @@
+import Link from "next/link";
+
+import BasePanel from "@/components/Sidebar/panels/BasePanel";
+import getQueue from "@/lib/playback/getQueue";
+
+const Queue = async () => {
+  const { queue } = await getQueue();
+
+  // We need to disable this rule because we're checking for null
+  // values in the next line, but TypeScript doesn't know that.
+  // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
+  if (!queue)
+    return (
+      <BasePanel title="Queue" sidebarId="queue">
+        <p>Nothing is playing!</p>
```

Comment:
<img width="260" alt="Screen Shot 2023-04-02 at 9 17 32 PM" src="https://user-images.githubusercontent.com/77420992/229390966-21f72ce4-32b3-4ea9-9c7f-8a314c9e2f04.png">

When Spotify has yet to connect no "Nothing is playing" message shows